pkgs/development/tools/zld/default.nix

```diff
@@ -2,7 +2,7 @@
 
 stdenv.mkDerivation rec {
   pname = "zld";
-  version = "1.3.3";
+  version = "1.3.4";
   src = fetchzip {
     url = "https://github.com/michaeleisel/zld/releases/download/${version}/zld.zip";
     sha256 = "0qb4l7a4vhpnzkgzhw0jivz40jr5gdhqfyynhbkhn7ryh5s52d1p";
```

Comment:
You also must update the hash.

Reply:
whoops, sorry about that - I've updated and pushed.